guides/products-and-variants/product-properties.md

```diff
@@ -0,0 +1,39 @@
+# Product properties
+
+Product properties belong to the `Spree::ProductProperty` model. They track
+individual attributes for a product that would not apply to all of your
+products. Typically, product properties would be used for additional product
+information.
+
+As an example, you might see a list of product properties for a limited edition
+t-shirt as a table on its product page:
+
+| Property name | Property value   |
+|---------------|------------------|
+| Fit           | Tapered          |
+| Manufacturer  | American Apparel |
+| Material      | 100% cotton      |
+
+You can retrieve the value for a property on a `Spree::Product` object by
+calling the `property` method on it and passing through that property's name:
+
+```shell
+Spree::Product.find(1).property("fit")
+=> "Tapered"
+```
+
+You can set a property on a product by calling the `set_property` method:
+
+```ruby
+Spree::Product.find(1).set_property("fit", "Tapered")
+```
+
+If this property doesn't already exist, a new `Property` instance with this name
+will be created.
+
+## Product properties are not option types
```

Comment:
This is a good point and needs further detail. When should I use a property, and when should I use an option type?